src/main/scala/rocket/ALU.scala

```diff
@@ -4,13 +4,14 @@
 package freechips.rocketchip.rocket
 
 import chisel3._
-import chisel3.util.{BitPat, Fill, Cat, Reverse}
+import chisel3.util.{BitPat, Fill, Cat, Reverse, PriorityEncoderOH, PopCount, MuxLookup}
 import org.chipsalliance.cde.config.Parameters
 import freechips.rocketchip.tile.CoreModule
+import freechips.rocketchip.util._
 
-class ALUFN {
-  val SZ_ALU_FN = 4
-  def FN_X    = BitPat("b????")
+object ALU {
```

Comment:
So question is here: the uop of ALU goes from 4 width to 5 width, a little increasing for all cores, but anyway in current rocket-chip methodology, we don't need to take it seriously.

Reply:
The high bit is useless when no B ext is enabled. It should be optimized away at synthesis